src/app/components/home/home.component.html

```diff
@@ -26,7 +26,7 @@ <h1>All projects in one place.</h1>
 <div class="row justify-content-center">
   <div class="col-12">
     <div class="search">
-      <input type="search" placeholder="Search source" [formControl]="searchControl" (keydown)="onKeyDownFunction($event)">
+      <input type="search" placeholder="search project" [formControl]="searchControl" (keydown)="onKeyDownFunction($event)" (keyup.enter)="onClickSearch()">
```

Comment:
The (keyup.enter)="onClickSearch()" is double since there is already a (keydown)="onKeyDownFunction($event)" which in the component checks if the pressed key is the enter key, and submits the search result, and redirects to the overview page

Reply:
Ah I did not read properly. The keydownFunction already checks for enter. I reverted my changes.